modules/client-configuration/pages/autoinst-profiles.adoc

```diff
@@ -54,15 +54,20 @@ This setting allows {productname} to automatically pick the latest distribution
 If you add new distributions later, {productname} will use the most recently created or modified.
 ====
 
-You can change the details and settings of your autoinstallation profiles by navigating to menu:Systems[Autoinstallation > Profiles] and clicking on the name of the profile you want to edit.
-
 [NOTE]
 ====
 Changing the Virtualization Type usually requires changes to the profile bootloader and partition options.
 This can overwrite your customization.
 Verify new or changed settings before saving them, by navigating to the [guimenu]``Partitioning`` tab.
 ====
 
+You can change the details and settings of your autoinstallation profiles by navigating to menu:Systems[Autoinstallation > Profiles] and clicking the name of the profile you want to edit.
+
+To adjust advanced {kickstart} configuration options, navigate to the menu:Autoinstallation Details[Advanced Autoinstallation Configuration] subtab.
```

Comment:
I was referring to the first ˋkickstartˋ macro. You get there via a button which is called ˋAdvanced Optionsˋ 
So I am not sure about your change.

Reply:
Oh, has the navigation changed? Can you give me a screenshot that shows the menu and tabs? (Or update refhead?)

Reply:
Here is the start page of Provisioning

![image](https://user-images.githubusercontent.com/1038917/98126104-812c2580-1eb5-11eb-9833-7f8a76844e63.png)

You can see the button `Advanced Configuration`. When you click on it, you come to this page:

![image](https://user-images.githubusercontent.com/1038917/98126241-a5880200-1eb5-11eb-99d5-5af900f267f5.png)

Here the title of the content box has changed: It is now called "Advanced Autoinstallation Configuration". 
In the past it was named "Advanced Kickstart Configuration".
This page looks a bit different when the system is a traditional managed client. Than we have additional configuration options. But the title has also changed.

![image](https://user-images.githubusercontent.com/1038917/98126806-4676bd00-1eb6-11eb-8030-e26b0e18706d.png)

Reply:
Ah ha, I understand now, I was navigating to the page quite differently. Updated, please check :)